packages/yew/src/virtual_dom/vcomp.rs

```diff
@@ -181,6 +183,13 @@ trait Mountable {
         next_sibling: NodeRef,
     ) -> Box<dyn Scoped>;
     fn reuse(self: Box<Self>, node_ref: NodeRef, scope: &dyn Scoped, next_sibling: NodeRef);
+
+    #[cfg(feature = "ssr")]
+    fn render_to_string<'a>(
+        &'a self,
+        w: &'a mut String,
```

Comment:
Let's have a more descriptive name than `w`

Reply:
This is a common convention that is used in `std`.

https://doc.rust-lang.org/std/macro.writeln.html

`buf` -> Buffer
`s` -> String
`w` -> Writer.

Reply:
I didn't know that convention